test/unit/mdc-tab-bar/mdc-tab-bar.test.js

```diff
@@ -227,13 +227,6 @@ test(`#adapter.notifyTabActivated emits the ${MDCTabBarFoundation.strings.TAB_AC
   td.verify(handler(td.matchers.contains({detail: {index: 66}})));
 });
 
-function setupMockFoundationTest(root = getFixture()) {
```

Comment:
The function is already defined above, and this copy is missing `const root = ...`.

Reply:
This copy had it as a parameter instead, but I guess if nothing was ever specifying that parameter it's fine